client/components/InternalNoteLabel/style.scss

```diff
@@ -10,3 +10,11 @@
         margin-left: 0.6rem;
     }
 }
+
+.icon--yellow2 {
+    color: #d2a800 !important
```

Comment:
In the UI-Framework the `yellow2` and `highlight2` are assigned a variable for the colour not a hash value. These two classes should use the same, so that if the colour changes in the UI-Framework, Planning will automatically get the updated colour values too.

Import the UI-Framework colours file:
`@import '~superdesk-ui-framework/app/styles/variables/_colors.scss';`

And use the following variables instead of has values:
`icon--yellow2` = `$oldGold !important`
`icon--hightlight2` = `$deepPink !important`